doc/developers.rst

```diff
@@ -27,16 +27,22 @@ The following should be installed in order to build the documentation.
 
 Shortcut: ``pip install sphinx sphinx-gallery pillow matplotlib``
 
+Some dependencies are necessary to build the documentation (i.e. `sphinx`, `sphinx-gallery` and `matplotlib`). At the root project folder, you can simply run::
```

Comment:
you need to put another quote around sphinx and sphinx-gallery to avoid it being shown in italics